tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -0,0 +1,152 @@
+raw_fnirs.plot_psd(average=True)
+
+###############################################################################
+# Extract epochs
+# --------------
+#
+# The expected response is below 0.5 Hz, and the increase in activity around 1
+# Hz is due to heart rate. So apply a low pass filter to remove unwanted
+# activity.
+
+events, _ = mne.events_from_annotations(raw_fnirs, event_id={'1.0': 1,
+                                                             '2.0': 2,
+                                                             '3.0': 3})
+event_dict = {'Control': 1, 'Tapping/Left': 2, 'Tapping/Right': 3}
+fig = mne.viz.plot_events(events, event_id=event_dict,
+                          sfreq=raw_fnirs.info['sfreq'])
+fig.subplots_adjust(right=0.7)  # make room for the legend
+
+reject_criteria = dict(hbo=80e-6)
+tmin, tmax = -5, 15
+
+epochs = mne.Epochs(raw_fnirs, events, event_id=event_dict,
+                    tmin=tmin, tmax=tmax,
+                    reject=reject_criteria, reject_by_annotation=True,
+                    proj=True, baseline=(None, 0), preload=True,
+                    detrend=None, verbose=True)
+epochs.plot_drop_log()
+
+
+###############################################################################
+# View consistency of responses across trials
+# -------------------------------------------
+#
+# Plot image
+
+epochs['Tapping'].plot_image(combine='mean', vmin=-30, vmax=30)
+
+###############################################################################
+
+epochs['Control'].plot_image(combine='mean', vmin=-30, vmax=30)
+
+
+###############################################################################
+# View consistency of responses across trials
+# -------------------------------------------
+#
+# Plot image
+
+# How do I make all these color maps on the same scale?
+fig, (cont, tapping) = plt.subplots(nrows=2, ncols=2, figsize=(15, 6))
+epochs['Control'].average().plot_image(axes=[cont[0], tapping[0]], show=False)
+epochs['Tapping'].average().plot_image(axes=[cont[1], tapping[1]], show=False)
+fig.show()
+
+
+###############################################################################
+# Plot standard fNIRS response image
+# ----------------------------------
+#
+# Plot image
+
+
+evoked_dict = {'HbO': epochs['Tapping'].average(picks='hbo'),
+               'HbR': epochs['Tapping'].average(picks='hbr')}
+
+# Rename channels until the encoding of frequency issue is fixed
+picks = _picks_to_idx(evoked_dict['HbO'].info, 'fnirs')
+for ii in picks:
+    evoked_dict['HbO'].rename_channels({
+        evoked_dict['HbO'].info['chs'][ii]['ch_name']:
+            evoked_dict['HbO'].info['chs'][ii]['ch_name'][:-4]})
+picks = _picks_to_idx(evoked_dict['HbR'].info, 'fnirs')
+for ii in picks:
+    evoked_dict['HbR'].rename_channels({
+        evoked_dict['HbR'].info['chs'][ii]['ch_name']:
+            evoked_dict['HbR'].info['chs'][ii]['ch_name'][:-4]})
+
+mne.viz.plot_compare_evokeds(evoked_dict, combine="mean", ci=0.95)
```

Comment:
What am I doing wrong that makes the confidence intervals not appear? They show in the `plot_image` calls above.

Reply:
The `plot_image` call operates on epochs, so there are multiple observations from which to compute the confidence band. But with `plot_compare_evokeds` you have a single evoked per condition. You can fake it by passing lists of evoked objects as the dict values, e.g. via `epochs.iter_evoked` (which is how the confidence bands in the image plot are done)

Reply:
Thanks. I guess its most appropriate for me to select a single channel for this example and show the response with CIs on it. I will get to that in the next days. Appreciate your help.